core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -440,6 +440,26 @@ public class RestConfig extends AbstractConfig {
           + "Default is false.";
   private static final boolean DOS_FILTER_MANAGED_ATTR_DEFAULT = false;
 
+  private static final String SERVER_CONNECTION_LIMIT = "server.connection.limit";
+  private static final String SERVER_CONNECTION_LIMIT_DOC =
+      "Limits the number of active connections on that server to the configured number. Once that "
+          + "limit is reached further connections will not be accepted until the number of active "
+          + "connections goes below that limit again. Active connections here means all already "
+          + "opened connections plus all connections that are in the process of being accepted. "
+          + "If the limit is set to a non-positive number, no limit is applied. Default is 0.";
```

Comment:
Is this quite right? Looking at the setting code above, there is no limit set if the default of 0 is used too (not just if a negative number is used)

Reply:
Or I guess it technically is correct if 0 is considered non-positive.... But I think that requires a little processing when reading the line that I think people will miss.